ui/src/session.ts

```diff
@@ -191,7 +191,7 @@ export const addSeed = async (seed: string): Promise<boolean> => {
   // running async remote validations in in the background. If we remove the
   // async then the seed input form will have to be submitted twice to take any
   // effect.
-  await seedValidation.validate(seed);
+  seedValidation.validate(seed);
```

Comment:
Is this safe, given the comment above?

Reply:
Removing this wasn’t fully intentional. I’ve reverted it in https://github.com/radicle-dev/radicle-upstream/pull/1215/commits/2c8bc6d0b24a6a829e85863b12ddc7f7f908dc94